qiskit/converters/ast_to_dag.py

```diff
@@ -161,7 +164,7 @@ def _process_bit_id(self, node):
         if node.type == "indexed_id":
```

Comment:
Is `indexed_id` an indexed identity gate or does id stand for identifier? Should this be changed to `indexed_i`?

Reply:
This is an indexed node used in the dag.